privacyidea/lib/policy.py

```diff
@@ -1242,6 +1254,12 @@ def get_static_policy_definitions(scope=None):
                                             "configuration."),
                                   "group": GROUP.SYSTEM,
                                   'mainmenu': [MAIN_MENU.CONFIG]},
+            ACTION.SYSTEMREAD: {'type': 'bool',
+                                "desc": _("Admin is allowed to read "
+                                          "basic system configuration "
```

Comment:
basic system configuration ~~configuration~~ :)

Reply:
OK. Will do. Thanks!

Reply:
done in 003f1a44
(might be a good idea to resolve conversations)